### PR TITLE
Change JSON files to JS

### PR DESCRIPTION
**Pull request by Xyfi**
In order to not have to copy json files when creating a publish build it's easier to have JS files instead of JSON files because they will be picked up by babel.

## Test instructions
* Run `grunt publish`
* Inspect `dist` folder and see if the language files are there (de.js, it.js, en.js and nl.js).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,11 +7,11 @@ We will follow [Semantic Versioning](http://semver.org/) from version 2 and onwa
...
-* Added `Excited` to exception -ed verbs. #1269
-* Changed testing framework to `jest`. #1266
-* Added `release` script in `.travis.yml` for publishing to npm. #1265
-* Added `code` tags to filter. #1250
-* Added `pre` tags to filter, props [chrisboo](https://github.com/chrisboo). #1258
+* Adds `Excited` to exception -ed verbs. #1269
+* Adds `release` script in `.travis.yml` for publishing to npm. #1265
+* Adds `code` tags to filter. #1250
+* Adds `pre` tags to filter, props [chrisboo](https://github.com/chrisboo). #1258
+* Switched testing framework to `jest`. #1266
```
Comment (IreneStr) [inline CHANGELOG.md]: Switches

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,11 +7,11 @@ We will follow [Semantic Versioning](http://semver.org/) from version 2 and onwa
 
 ## 1.23.0 October 31st, 2017
 * Errors now give a score of -1. `ScoreToRating` interpreter will convert `-1` to `error`. #1272
-* Added `Excited` to exception -ed verbs. #1269
-* Changed testing framework to `jest`. #1266
-* Added `release` script in `.travis.yml` for publishing to npm. #1265
-* Added `code` tags to filter. #1250
-* Added `pre` tags to filter, props [chrisboo](https://github.com/chrisboo). #1258
+* Adds `Excited` to exception -ed verbs. #1269
```
Comment (IreneStr) [inline CHANGELOG.md]: for passive voice assessment

File: CHANGELOG.md [hunk 3]
```diff
@@ -7,11 +7,11 @@ We will follow [Semantic Versioning](http://semver.org/) from version 2 and onwa
...
+* Adds `release` script in `.travis.yml` for publishing to npm. #1265
```
Comment (IreneStr) [inline CHANGELOG.md]: `to` ipv `in`

File: CHANGELOG.md [hunk 4]
```diff
@@ -7,11 +7,11 @@ We will follow [Semantic Versioning](http://semver.org/) from version 2 and onwa
...
+* Adds `code` tags to filter. #1250
```
Comment (IreneStr) [inline CHANGELOG.md]: Which filter? Not very descriptive.

File: CHANGELOG.md [hunk 5]
```diff
@@ -7,11 +7,11 @@ We will follow [Semantic Versioning](http://semver.org/) from version 2 and onwa
...
+* Adds `pre` tags to filter, props [chrisboo](https://github.com/chrisboo). #1258
```
Comment (IreneStr) [inline CHANGELOG.md]: Same here

**Comment by IreneStr** [comment]
CR 🚧  The json format is used for a reason. Please discuss with @atimmer

File: CHANGELOG.md (hunk 4 above)
Comment (IreneStr) [reply CHANGELOG.md]: Adds a filter to exclude `code` tags and their enclosed content from the content analysis assessments.

File: CHANGELOG.md (hunk 5 above)
Comment (IreneStr) [reply CHANGELOG.md]: Adds a filter to exclude `pre` tags and their enclosed content from the content analysis assessments.

**Comment by Xyfi** [comment]
Closing in favour of #1277

**Pull request closed without merging**